src/views/Drawer/DrawerView.js

```diff
@@ -63,6 +63,12 @@ export default class DrawerView<T: *> extends PureComponent<void, Props, void> {
       const { routes, index } = nextProps.navigation.state;
       if (routes[index].routeName === 'DrawerOpen') {
         this._drawer.openDrawer();
+      } else if (routes[index].routeName === 'DrawerToggle') {
+        if (this._drawer.state.drawerShown) {
+          this.props.navigation.navigate('DrawerClose');
```

Comment:
To keep it consistent with DrawerOpen/DrawerClose how about using `this._drawer.closeDrawer();` and `this._drawer.openDrawer();` instead?

Reply:
Sorry, missed this before approving

Reply:
@spencercarli - using `this._drawer.closeDrawer();` here won't update the state, or fire off the events. 

 If we switch this out it will animate into place, not fire any events, and then suddenly hide due to not being in that new state.

This way solves it.